src/utils/logger.ts

```diff
@@ -1,24 +1,24 @@
-import chalk from "chalk";
+import chalk, { Chalk } from "chalk";
 import { format, createLogger, transports } from "winston";
 
 import { hasColor } from "./helpers.js";
```

Comment:
Also please make sure you import types as actual types.
<img width="671" alt="image" src="https://github.com/matter-labs/zksync-cli/assets/47187316/9d11dba9-8964-447d-bb8b-ed2a997cbe9a">

```suggestion
import chalk from "chalk";
import { format, createLogger, transports } from "winston";

import { hasColor } from "./helpers.js";

import type { Chalk } from "chalk";
```